synfig-studio/src/gui/app.cpp

```diff
@@ -542,6 +543,11 @@ class Preferences : public synfigapp::Settings
 				value=App::sequence_separator;
 				return true;
 			}
+			if(key=="number_of_threads")
+			{
+				value=App::number_of_threads;
```

Comment:
@SushantAA  maybe you should convert this integer into string before assign to value?

Reply:
It should be a string :grimacing:
@rodolforg should the initial setting of threads should be 2 or 12. currently, it's 2 but I think 12 will be better

Reply:
Look few lines above: use `etl::strprintf()` to 'convert' the `App::number_of_threads`.

IIRC, you make it to be the maximum threads as default.

Reply:
https://github.com/synfig/synfig/pull/2023